stream/statelib/src/main/java/org/apache/bookkeeper/statelib/impl/rocksdb/checkpoint/dlog/DLCheckpointStore.java

```diff
@@ -98,6 +100,9 @@ public OutputStream openOutputStream(String filePath) throws IOException {
         try {
             DistributedLogManager dlm = namespace.openLog(
                 filePath);
+            LedgerMetadata metadata = new LedgerMetadata();
+            metadata.setApplication(Constants.LEDGER_METADATA_APPLICATION_STREAM_STORAGE);
+            metadata.setComponent("checkpoint-store");
```

Comment:
this looks like a dead store,
you are not using this variable

Reply:
passed to the openAsyncLogWriter call

Reply:
ok